website/src/segment.css.ts

```diff
@@ -167,3 +167,15 @@ export const infoIcon = style({
 })
 
 export const linkSvg = style({ fill: colors.link })
+
+export const fillerLine = style([
+  {
+    "@media": {
+      [mediaQueries.medium]: {
+        borderBottom: "1px solid black",
+        padding: 0,
+        ...margin(vspace.one, "3rem", vspace.one, 0),
+      },
+    },
+  },
+])
```

Comment:
We might want to run `prettier` on a commit hook or something but for now, would you mind turning on "format on save" in your editor? If you're using VSCode it has built-in support for `prettier`. That'll automatically fix this line ending thing and some extra spaces you have elsewhere.